### PR TITLE
Move entity classes to EF-Independent project

### PR DESCRIPTION
**Pull request by ckalan**
I've moved the entity classes to a separate project so that people can reference them in their model or domain projects without any requirement to reference EntityFramework. This way one can use the same model in the future with EF Core.

### CODE REVIEW

**Comment by ckalan** [comment]
@mrahhal If you are OK with the PR, could you please upload the Model nuget package so that we can reference it in our projects?

File: src/MR.AspNet.Identity.EntityFramework6.Model/MR.AspNet.Identity.EntityFramework6.Model.csproj [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <Version>3.0.4</Version>
```
Comment (mrahhal) [inline src/MR.AspNet.Identity.EntityFramework6.Model/MR.AspNet.Identity.EntityFramework6.Model.csproj]: There is a common.props file that I import in all csprojs that let me share lots of those values, and also reference a single version. So please check the other csprojs on how to edit this one to take advantage of common.props. You'll still need to add the `TargetFramework` element above to override that.

File: build.cake [hunk 2]
```diff
@@ -12,7 +12,11 @@ Task("Clean")
 {
 	if (DirectoryExists("./artifacts"))
 	{
-		DeleteDirectory("./artifacts", true);
+		DeleteDirectory("./artifacts", new DeleteDirectorySettings {
+		Recursive = true,
+		Force = true
+	});
```
Comment (mrahhal) [inline build.cake]: Some indentation on the last 3 lines here please. And remove the line trailing line after this.

**Review by mrahhal — CHANGES_REQUESTED** [review]
Sounds like a good idea.

In addition to the review comments I added that require changes, could you also rename the project from "Model" to "Models"? I'd like to follow this convention. (Make sure the folder's name is renamed too).

After those changes, I'll start releasing this as a package.

**Comment by ckalan** [comment]
I've made some changes based on your requests. Could you please check?

File: src/MR.AspNet.Identity.EntityFramework6.Model/MR.AspNet.Identity.EntityFramework6.Model.csproj [hunk 3]
```diff
@@ -1,16 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="../../build/version.props" />
```
Comment (mrahhal) [inline src/MR.AspNet.Identity.EntityFramework6.Model/MR.AspNet.Identity.EntityFramework6.Model.csproj]: "common.props" please! It includes "version.props" and adds stuff like "PackageLicenseUrl" and the like.

**Review by mrahhal — APPROVED** [review]
Ok looks good. I can change the project's name myself as I need to do some edits in any case. I'll merge this with another branch to do my edits.

**Comment by mrahhal** [comment]
Fixing an issue with netstandard tfm usage. Package "MR.AspNet.Identity.EntityFramework6.Models" should be ready by today.

**Comment by ckalan** [comment]
I didn't have time to fix those today. I see you've already merged it. Thanks!

**Comment by mrahhal** [comment]
Released as `3.0.5`, thanks a lot!

**Pull request merged**